packages/api/tests/lambdas/test-sf-semaphore-down.js

```diff
@@ -37,6 +40,43 @@ const createCloudwatchEventMessage = ({
   }
 });
 
+const createCloudwatchPackagedEventMessage = ({
+  status,
+  queueName,
+  source = sfEventSource
+}) => ({
+  source,
+  detail: {
+    status,
+    output: JSON.stringify({
+      cumulus_meta: {
+        execution_name: randomString(),
+        queueName
+      },
+      replace: {
+        Bucket: 'cumulus-sandbox-testing',
+        Key: 'stubbedKey'
+      }
+    })
+  }
+});
+
+
+const packagedEventReturn = ((queueName) => (
+  {
```

Comment:
I could probably refactor this to just update the packagedEvent object.   I was wavering on this being explicit given it's a unit test.   🤔